src/main/java/org/myrobotlab/service/Runtime.java

```diff
@@ -1684,7 +1694,7 @@ static public void stopCli() {
   // the current concept is ok - but it does not work ..
   // make it work if necessary prefix everything by -agent-<...>
   // FIXME - implement --help -h !!! - handle THROW !

```

Comment:
seems like this FIXME is done now.. clean up the comment.  (also, what's the Command annotation? is that an example usage syntax?)

Reply:
ok removing ...
Command annotation is part of picocli its whats printed out first on help